### PR TITLE
Scrolling payment button bug

### PR DESCRIPTION
**Pull request by nauaros**
## Summary
Fix scrolling bug with pay button.

## Tested scenarios

**Fixed issue**:  https://github.com/Adyen/adyen-ios/issues/643

### CODE REVIEW

File: Demo/Configuration.swift [hunk 1]
```diff
@@ -37,7 +37,7 @@ internal enum ConfigurationConstants {
 
     static let applePayMerchantIdentifier = "{YOUR_APPLE_PAY_MERCHANT_IDENTIFIER}"
 
-    static let merchantAccount = "{YOUR_MERCHANT_ACCOUNT}"
+    static let merchantAccount = "TestMerchantCheckout"
```
Comment (descorp) [inline Demo/Configuration.swift:40]: This is not needed

Comment (nauaros) [reply Demo/Configuration.swift]: fixed

File: Adyen/Extensions/NSLayoutConstraint+Extensions.swift [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+
+import Foundation
+
+extension NSLayoutConstraint {
+
+    class func activateConstrainst(_ constraints: [NSLayoutConstraint]) {
+        constraints.forEach { constraint in
+            if let view = constraint.firstItem as? UIView {
+                view.translatesAutoresizingMaskIntoConstraints = false
+            }
```
[the remark this answers is not in the document]
Comment (nauaros) [reply Adyen/Extensions/NSLayoutConstraint+Extensions.swift]: Now we set `translatesAutoresizingMaskIntoConstraints` in the anchor methods

File: Demo/Configuration.swift (hunk 1 above)
Comment (mohammedDehairy) [inline Demo/Configuration.swift:40]: please revert this :-D

**Review by mohammedDehairy — APPROVED** [review]

**Review by descorp — APPROVED** [review]

**Pull request merged**